test/core/test_lwt.ml

```diff
@@ -1801,7 +1801,56 @@ let async_tests = suite "async" [
       !saw = Some Exception)
   end;
 ]
-let suites = suites @ [async_tests]
+let dont_wait_tests = suite "dont_wait" [
+  test "fulfilled" begin fun () ->
+    let f_ran = ref false in
+    Lwt.dont_wait (fun () -> f_ran := true; Lwt.return ()) (fun _ -> ());
+    later (fun () -> !f_ran = true)
+  end;
+
+  test ~sequential:true "f raises" begin fun () ->
```

Comment:
This test doesn't need to be sequential. That's probably a leftover from using `async` tests as a template, where the test corresponding to this one mutates the global `async_exception_hook` ref. This doesn't happen in `dont_wait` tests.

Reply:
The same goes for the other tests in this suite with `~sequential:true`.